test/spanner/config/validator_test.exs

```diff
@@ -17,10 +17,68 @@ defmodule Spanner.Config.Validator.Test do
     }
   end
 
+  defp enforcing_config do
+    %{"name" => "foo",
+      "version" => "0.0.1",
+      "commands" => %{
+        "bar" => %{
+          "executable" => "/bin/bar",
+          "enforcing" => true}}}
+  end
```

Comment:
You could do, 
```
...
put_in(minimal_config, ["commands", "bar"], %{"executable" => "/bin/bar", "enforcing" => true})
...
```

Doesn't really matter, but it would fit more with the pattern of the other tests.